File: task-03/Ruby/prime_number.rb
print "Enter a number:"
num = gets.chomp.to_i
if num < 2
  puts "No Prime"
else
  for i in 2..num
    count = 0
    for j in 2..i-1
      if i%j == 0
        count= 1
        break
      else
        count= 0
      end
    end
    if (count== 0)
      puts "#{i} \t"
    end
  end
end

File: task-03/Java/prime_number.java
import java.util.Scanner;
public class main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number: \n");
        int n = sc.nextInt();
                System.out.println("The prime numbers are:");
        
        for(int i = 2; i <= n; i++){
            int count = 0;
            for(int j = 2; j*j <= i; j++){
                if(i % j == 0){
                    count = 1;
                    break;
                }
            }
            if(count == 0){
            System.out.print(i + " ");
                
            }
        }
    }
}

